tide_landing_page.install

```diff
@@ -1312,3 +1312,49 @@ function tide_landing_page_update_8033() {
   $config_read = _tide_read_config($json_field, $config_location, TRUE);
   $config_storage->write($json_field, $config_read);
 }
+
+/**
+ * Installs new navigation_card paragraph type.
+ */
+function tide_landing_page_update_8034() {
+  $configs = [
+    'paragraphs.paragraphs_type.navigation_card' => 'paragraphs_type',
+    'field.storage.paragraph.field_nav_card_display_style' => 'field_storage_config',
+    'field.field.paragraph.navigation_card.field_paragraph_title' => 'field_config',
+    'field.field.paragraph.navigation_card.field_paragraph_summary' => 'field_config',
+    'field.field.paragraph.navigation_card.field_paragraph_media' => 'field_config',
+    'field.field.paragraph.navigation_card.field_paragraph_link' => 'field_config',
+    'field.field.paragraph.navigation_card.field_customise' => 'field_config',
+    'field.field.paragraph.navigation_card.field_nav_card_display_style' => 'field_config',
+    'core.entity_view_display.paragraph.navigation_card.default' => 'entity_view_display',
+    'core.entity_form_display.paragraph.navigation_card.default' => 'entity_form_display',
+  ];
+  module_load_include('inc', 'tide_core', 'includes/helpers');
+  $config_location = [drupal_get_path('module', 'tide_landing_page') . '/config/install'];
+  // Check if field already exported to config/sync.
+  foreach ($configs as $config => $type) {
+    $config_read = _tide_read_config($config, $config_location, TRUE);
+    $storage = \Drupal::entityTypeManager()->getStorage($type);
+    $config_entity = $storage->createFromStorageRecord($config_read);
+    $config_entity->save();
```

Comment:
Can you test if this works? The comment says "Check if field already exported to config/sync." but the code doesn't looks like it is doing that.

Reply:
Hey @asarfara 
`_tide_read_config` actually checks that. So setting it _True_ will prioritize the config/sync location and if it is there, it will return the configs from there.